recipes/crowdsource/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "crowdsource" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 641fb3678a433fe6b017589808e74dab0fcbe9cc514e3d17b58b1a92aec763b5
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - nodejs
+  run:
+    - python >=3.7
+    - cufflinks >=0.17.0
+    - IPython >=7.4.0
+    - jinja2 >=2.10
+    - numpy >=1.17.2
+    - pandas >=0.25.0
+    - perspective-python >=0.4.6
+    - psycopg2 >=2.7.6.1
+    - scikit-learn >=0.21.3
+    - scipy >=1.2.1
+    - sqlalchemy >=1.3.0
+    - tornado-sqlalchemy-login >=0.1.0
```

Comment:
It looks like `tornado-sqlalchemy-login` and `perspective-python` needs to be added to conda-forge. I also think `cufflinks` might need to be referred to as `python-cufflinks`